sources/contract.spec.ts

```diff
@@ -529,7 +529,129 @@ describe("JettonMinter", () => {
 
         expect(await deployerJettonWallet.getJettonBalance()).toEqual(initialJettonBalance);
     });
+    describe('Bounces', () => {
+        // This is borrowed from stablecoin, and is not implemented here.
+        // Should it be implemented?
+        it('minter should restore supply on internal_transfer bounce', async () => {
+            const deployerJettonWallet    = await userWallet(deployer.address);
+            const mintAmount = BigInt(getRandomInt(1000, 2000));
+            const mintMsg = beginCell().store(storeMint({$$type: "Mint", amount: mintAmount, receiver: deployer.address})).endCell();
+            //const mintMsg    = JettonMinter.mintMessage(jettonMinter.address, deployer.address, mintAmount,toNano('0.1'), toNano('0.1'));
```

Comment:
there should be no commented out code; if you need it for some reason, please explain it as a comment